protocols/raft/src/main/java/io/atomix/protocols/raft/proxy/impl/DefaultRaftProxy.java

```diff
@@ -68,6 +69,7 @@
   private volatile RaftProxyListener proxyListener;
   private volatile RaftProxyInvoker proxyInvoker;
   private volatile RaftProxyState state;
+  private final Consumer<NodeId> leaderChangeListener = this::onLeaderChange;
 
   public DefaultRaftProxy(
```

Comment:
This is called DiscreteRaftProxyClient in 2.0, correct?

Reply:
Yep